Documentation/gettingstarted/kubeproxy-free.rst

```diff
@@ -706,6 +706,31 @@ and therefore not affecting any application pod ``bind(2)`` requests anymore. In
 order to opt-out from this behavior in general, this setting can be changed for
 expert users by switching ``global.nodePort.bindProtection`` to ``false``.
 
+.. _Configuring Maps:
+
+Configuring BPF map sizes
+*************************
+
+For high-scale environments, Cilium's BPF maps can be configured to have higher
+limits on the number of entries. Overriding helm options can be used to tweak
+these limits.
+
+To increase the number of entries in Cilium's BPF LB service, backend and
+affinity maps consider overriding ``global.bpf.lbMapMax`` helm option.
+The default value of this LB map size is 65536.
+
+.. note::
+
+   Scaling size of Cilium's BPF LB map comes with a limitation that the Reverse NAT
```

Comment:
This might confuse users - for each service we create a surrogate entry which doesn't point to any backend, but it is used to determine a number of backends (for the second lookup). So, for a service w/o any backend we have 1 entry with the same rev NAT ID, while w/ N backends - N + 1.

Reply:
As discussed on slack with @brb, since revNAT ID limit will be hit later than the LB map max size(unless user runs services without any backend) it might make sense to remove this note. @borkmann what are your thoughts on this?

Reply:
Yes, ACK for removing this note.